services/search/pkg/search/service.go

```diff
@@ -246,7 +246,12 @@ func (s *Service) searchIndex(ctx context.Context, req *searchsvc.SearchRequest,
 			return nil, err
 		}
 
-		gpRes, err := gatewayClient.GetPath(ctx, &provider.GetPathRequest{
+		ownerCtx, err := getAuthContext(&user.User{Id: space.Owner.Id}, s.gatewaySelector, s.secret, s.logger)
```

Comment:
this might not work for project spaces, because they don't have an owner ...
oh wait ... we have a these type space_owner users 🤔